src/App.module.css

```diff
@@ -8,6 +8,60 @@ label {
   gap: 2px;
 }
 
+.header {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: space-between;
+  width: auto;
+  height: fit-content;
+  --min-height: 24px; /* Pass to children, namely for vertical divider formatting*/
```

Comment:
Rambling on variables, since I believe you asked: I think of this use case for CSS variables (ensuring that properties within multiple selectors have the same value) as kinda analogous to CAPITAL_SNAKE_CASE constants at the top of code files. I'd therefore try to ensure they were used similarly: placed near the top of the file, and accessible in the global scope. "Global scope" in CSS files I guess would mean "defined under the class of the top-most parent element..." though given the contents of this file it's reasonable to call `.header` high enough in the hierarchy. Regardless, my instinct would be to place it at the top of the class and visually separate it a bit.